django/db/backends/postgresql/base.py

```diff
@@ -543,6 +764,11 @@ def pg_version(self):
         with self.temporary_connection():
             return self.connection.info.server_version
 
+    @cached_property
+    async def apg_version(self):
```

Comment:
```python
In [10]: class Test:
    ...:     @cached_property
    ...:     async def some(self):
    ...:         print('calculate ...')
    ...: 

In [11]: test = Test()

In [12]: await test.some
calculate ...

In [13]: await test.some
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[13], line 1
----> 1 await test.some

RuntimeError: cannot reuse already awaited coroutine
```

Probably we have problems with this solution

Reply:
we can try something like that

```python
        async def pg_version(self):
            if self._pg_version is not None:
                return self._pg_version

            async with self.temporary_connection():
                self._pg_version = self.connection.info.server_version
                return self._pg_version
```

Reply:
Thank you for catching that! I've fixed with your solution